packages/vega-parser/src/config.js

```diff
@@ -1,10 +1,20 @@
-var defaultFont = 'sans-serif',
+import {scheme} from 'vega-scale';
+import {scaleLinear} from 'd3-scale';
+import {range} from 'd3-array';
+import {interpolateLab} from 'd3-interpolate';
+
+const tab10 = scheme('tableau10');
+const defaultFont = 'sans-serif',
     defaultSymbolSize = 30,
     defaultStrokeWidth = 2,
-    defaultColor = '#4c78a8',
-    black = '#000',
-    gray = '#888',
-    lightGray = '#ddd';
+    blue = {signal: 'colors.blue'},
+    lightgray = {signal: 'colors.gray8'},
```

Comment:
It's worth noting that lightgray was `#ddd`, which is equivalent to `rgb(221,221,221)`.
Changing to `gray8`, which is rgb(226, 226, 226), would be a breaking change as all Vega plots would look different.

To avoid such breaking changes, one potential compromise is to do 8 steps (#000, #222, #444, #666, ..., #fff), which will include both #888 and #ddd that we previous used.  

We could consider 16 steps as well, but I think 10 is already too many, so I'd prefer 8. 

Since we're not using these grays to encode quantitative values, it might be less important to make them uniformly sampled from LAB.

Reply:
I’m okay with this. The reason for 11 stops was to have them represent percentages. I can change to your suggestions if @jheer agrees with it.

Reply:
Which stops do you want @kanitw?

`#000, #222, #444, #666, #888, #aaa, #ccc, #ddd, #fff` (#ccc and #ddd are closer than the others).

Reply:
Right, I'm a bit dumb here.  There are actually only 15 steps from 0, 1, 2, ... , E, F.  

Hmm

Reply:
That's what I did in the last few commits. It's a lot of colors but at least we have all the stops we have used so far.